docs/getting_started/installation_guide/platforms/custom.rst

```diff
@@ -186,53 +182,55 @@ The following components are supported by TLS version 1.2+:
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| resourcemgr       | yarn       | hadoop    | 8032                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| nodemanager       | yarn       | hadoop    | 8042                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| hopsworks         | glassfish  | glassfish | 443/8181), 4848      | Yes, only for 443/8181.                                        |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| elasticsearch     | elastic    | elastic   | 9200, 9300           | Not required                                                   |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| logstash          | elastic    | elastic   | 5044-5052, 9600      | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| filebeat          | elastic    | elastic   |        N/A           | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| kibana            | kibana     | elastic   | 5601                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| ndbmtd            | mysql      | mysql     | 10000                | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| mysqld            | mysql      | mysql     | 3306                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| ndb_mgmd          | mysql      | mysql     | 1186                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| hiveserver2       | hive       | hive      | 9084-9085            | No                                                             |
```

Comment:
we only use 9085 these days. and Yes you need to open the port to the externally if you want to use SageMaker or Python integration.

Reply:
Is that due to PyHive or something else?

Reply:
Yes it's PyHIve